cid-redirects.json

```diff
@@ -671,11 +671,7 @@
   "/07Sumo-Logic-Apps/01Amazon_and_AWS/AWS_Elastic_Load_Balancing_-_Classic": "/docs/integrations/amazon-aws/elastic-load-balancing-classic",
   "/07Sumo-Logic-Apps/01Amazon_and_AWS/AWS_Elastic_Load_Balancing_-_Classic/01-Collect-logs-for-the-AWS-Elastic-Load-Balancing-App": "/docs/integrations/amazon-aws/elastic-load-balancing-classic",
   "/07Sumo-Logic-Apps/01Amazon_and_AWS/AWS_Elastic_Load_Balancing_-_Classic/AWS-Elastic-Load-Balancer---Classic-App-Dashboards": "/docs/integrations/amazon-aws/elastic-load-balancing-classic",
-  "/07Sumo-Logic-Apps/01Amazon_and_AWS/AWS_Elastic_Load_Balancing_Metrics": "/docs/integrations/amazon-aws/elastic-load-balancing-metrics",
```

Comment:
@JV0812 , we don't delete cid-redirects links without redirecting to something new. Those legacy links (e.g., `/07Sumo-Logic-Apps/01Amazon_and_AWS/AWS_Elastic_Load_Balancing_Metrics`) are still out there and users will get 404s for all of them.

Reply:
Thanks Kim, I have updated the URL to the closest equivalent content.

Reply:
thank you!

Reply:
Thank you! So it looks like you've updated the legacy redirects, which is correct. However, you also need to write redirects from the deprecated doc URLs to the new doc URLs. Deprecated docs links exist elsewhere (Google search results, saved bookmarks, etc.). [Here's the instructions.](https://help.sumologic.com/docs/contributing/remove-doc/)

For example:
`"/docs/integrations/amazon-aws/elastic-load-balancing-metrics": "/docs/integrations/amazon-aws/elastic-load-balancer-app",`

Reply:
@JV0812 I've gone ahead and fixed these